### ISSUE TITLE
Accessiblity Issues on Team Edit form

### ISSUE THREAD

**Issue by shailpatels** [Submitty/Submitty#5957]
**Describe the bug**
- [ ] Issue 1: A number of textarea inputs do not have a label associated with them, same thing with the dropdown boxes.
Here is a screenshot I took useing WAVE:
![image](https://user-images.githubusercontent.com/12129065/96100101-fbaeea00-0ea1-11eb-89a8-6ca224f93888.png)

See this page for more information on adding labels to input elements:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

- [ ] Issue 2: The team edit history is composed of textarea inputs that are all disabled. As far as I can see the user is not able to edit the history through the edit form, if this is the case the history should be using an HTML table instead.

**To Reproduce**
1. Any team edit form, I looked at the ones under open team homework

**Comment by preetijangra15** [comment]
i can fix this
